ui5/eve7/lib/EveScene.js

```diff
@@ -40,9 +40,6 @@ sap.ui.define(['rootui5/eve7/lib/EveManager'], function(EveManager) {
 
       if (obj3d)
       {
-         // MT ??? why?, it can really be anything, even just container Object3D
-         obj3d._typename = "THREE.Mesh";
-
          // add reference to a streamed eve element to obj3d
```

Comment:
This `_typename` used by JSROOT.TGeoPainter. 
Normally JSROOT works with ROOT objects - all with `_typename` member.
Here I adding typename to the `three.js` object to simplify handling on JSROOT side. 
Please left it as it was before.

Reply:
OK, done ... thanks!

Reply:
But commit is still there - do you want to remove it?